Themis/ViewModels/Assessment/CodeEditorViewModel.swift

```diff
@@ -29,6 +29,10 @@ class CodeEditorViewModel: ObservableObject {
     @Published var showEditFeedback = false
     @Published var pencilMode = true
     @Published var feedbackForSelectionId = ""
+    @Published var feedbackSuggestions = [FeedbackSuggestion(srcFile: "/src/de/tum/themis/BubbleSort.java", fromLine: 1, toLine: 1),
+                                          FeedbackSuggestion(srcFile: "/src/de/tum/themis/Client.java", fromLine: 30, toLine: 40),
+                                        FeedbackSuggestion(srcFile: "/src/de/tum/themis/MergeSort.java", fromLine: 1, toLine: 15)]
```

Comment:
This is just mocked data, right? 

I'm just gonna comment on this so that you won't forget (which is unlikely) to change it later when we have it functional 😄

Reply:
As mentioned above: 
> I added some mock feedback suggestions for testing purposes.
Thank you anyways.